specifyweb/frontend/js_src/lib/components/FormEditor/fetchAllViews.ts

```diff
@@ -78,7 +78,7 @@ const augmentDatabaseView = (
       ? userInformation.availableCollections.find(
           ({ id }) => id === view.collectionId
         )?.collectionName
-      : undefined) ?? camelToHuman(view.viewsetLevel),
+      : undefined) ?? camelToHuman(`${view.viewsetLevel} ${view.viewsetName}`),
```

Comment:
(optional) could we put some punctuation between level and name?

i.e
`${view.viewsetLevel} > ${view.viewsetName}`
or the nicer
`${view.viewsetName} (${view.viewsetLevel})`